### ISSUE TITLE
Move short link button to a more discoverable location

### ISSUE THREAD

**Issue by georgf** [mozilla/telemetry-dashboard#423]
The short link button in the top right corner is hard to discover for people.
This probably affects `dist.html` and `evo.html` in `/new-pipeline`.

We would like to try to move it elsewhere and see if that improves discoverability.

![screen shot 2018-02-28 at 14 38 41](https://user-images.githubusercontent.com/407696/36807389-240f1bf6-1cc3-11e8-822e-064bc8ff1ac5.png)

**Comment by georgf** [comment]
Erin is looking into this.

**Comment by ecomerford** [comment]
@georgf @chutten I made a pull request for a simple update that moves the generator to the beginning of the button row.

I wanted to give it a name to draw more attention to it, but I can't figure out a good way to do so. I tried turning it into a button but that became an awkward button inside of a button. Plus, the generator made it larger than the other buttons in its row. I can't seem to find any other Bootstrap class types that would do the trick.

I suppose I could make it into a button, then disable it and try to adjust the CSS to make it the same size as the others, but that seems a bit strange. 

Any ideas on how to make that work?
